android/titanium/src/java/org/appcelerator/titanium/TiLaunchActivity.java

```diff
@@ -181,6 +186,7 @@ protected void onResume()
 		if (!hasLoadedScript) {
 			hasLoadedScript = true;
 			loadScript();
+			Log.i(TAG, "Launched in " + (System.currentTimeMillis() - TiApplication.START_TIME_MS) + " ms");
```

Comment:
This is probably a debug left over? Or maybe keep here but on debug level?

Reply:
Right. This should be removed since it's for debugging.

Especially since launching the app, backing-out, then re-launching will cause this to log again but with the old start time set upon cold start.